.yarnrc.yml

```diff
@@ -4,10 +4,12 @@ nodeLinker: node-modules
 
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
-    spec: '@yarnpkg/plugin-workspace-tools'
+    spec: "@yarnpkg/plugin-workspace-tools"
```

Comment:
Why change these? Is this a linter thing?

Reply:
Yes - the pre-commit hook runs these so I don't see them in the diff prior to committing.